integration/setup/config.go

```diff
@@ -0,0 +1,134 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2esetup/k8s"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	namespace       = "giantswarm"
+	tillerNamespace = "kube-system"
+)
+
+type Config struct {
+	Guest   *framework.Guest
+	Host    *framework.Host
+	K8s     *k8s.Setup
+	Release *release.Release
+	Logger  micrologger.Logger
+
+	// UseDefaultTenant defines whether the standard test setup should ensure the
+	// default tenant cluster. This is enabled by default. Most tests simply use
+	// the standard test tenant cluster. One exception is IPAM. There we launch
+	// multiple tenant clusters and do not want to make use of the default one.
+	UseDefaultTenant bool
```

Comment:
Lets remove this. 

Its not needed now and we can add it later when we add tests that use a tenant cluster.